docs/ldml/tr35-keyboards.md

```diff
@@ -1304,7 +1391,7 @@ Control characters, combining marks and whitespace in this attribute are escaped
 
 _Attribute:_ `to` (required)
 
-> This attribute represents the characters that are output from the transform. The output can contain more than one character, so you could have `<transform from="´A" to="Fred"/>`
+> This attribute represents the characters that are output from the transform. The output can contain more than one character, so you could have `<transform from="´A" to="Fred"/>`.
 
 Control characters, whitespace (other than the regular space character) and combining marks in this attribute are escaped using the `\u{...}` notation.
 
```

Comment:
Let's remove the platform names from these examples.

Reply:
good idea.